lib/puppet/type/runlevel.rb

```diff
@@ -29,16 +29,35 @@ def initialize(*args)
     end
   end
```

Comment:
where is the test for these changes

Reply:
Right here: https://github.com/simp/pupmod-simp-gdm/pull/34/files#diff-9253f3c9cbffa1500f41d4db2baa8664R29